libs/components/modals/src/lib/modules/modal/modal.component.ts

```diff
@@ -58,13 +58,29 @@ export class SkyModalComponent implements AfterViewInit, OnDestroy {
   @Input()
   public tiledBody: boolean | undefined;
 
-  public ariaDescribedBy: string;
+  /**
+   * @internal
+   */
+  @Input()
+  public set ariaDescribedBy(id: string | undefined) {
```

Comment:
I needed to be able to pass in `undefined` here so  I modified this to use a setter and then went ahead and changed `ariaLabelledBy` as well just for consistency.